network/message/init.go

```diff
@@ -1,9 +1,42 @@
 package message
 
+import (
+	"fmt"
+)
+
+var (
+	validationExclusionList = []string{TestMessage}
+)
+
 // init is called first time this package is imported.
 // It creates and initializes AuthorizationConfigs for each message type.
 func init() {
 	initializeMessageAuthConfigsMap()
+	validateMessageAuthConfigsMap(validationExclusionList)
+}
+
+func validateMessageAuthConfigsMap(excludeList []string) {
```

Comment:
can you add a comment explaining why this check is here

Reply:
https://github.com/onflow/flow-go/pull/3525/commits/5c2ce17c2000894e614422ede5eaee8417b35918